.env

```diff
@@ -1,4 +1,5 @@
 DEBUG=on
+PROD=off
```

Comment:
Perhaps this should instead be called `DUMMY_COMPILER`?

Reply:
I thought there may be other things we want to put behind this flag, like attempting to use other features that shouldn't work locally (permuter@home)

Reply:
I'd rather use separate flags for that. (also why would permuter@home not work in dev? how would we test it?)